companies/4care-gmbh.json

```diff
@@ -8,9 +8,10 @@
     ],
     "name": "4Care GmbH",
     "address": "Wissenschaftspark\nFraunhoferstr. 17\n24118 Kiel\nDeutschland",
-    "phone": "+49 431 881188",
+    "phone": "+49 431 881188a",
```

Comment:
🚫 **[recordlint]** <[pass-schema](https://github.com/datenanfragen/data#data-formats)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Record did not pass schema validation: “The value at /phone is an invalid string.”

Raw error:
```json
{
    "path": "$.phone",
    "pointer": "/phone",
    "message": "The value at /phone is an invalid string.",
    "data": "+49 431 881188a"
}
```